### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by MrWeatherf**

### CODE REVIEW

**Comment by MrWeatherf** [comment]
Flipping Australia "flag" (uses official national colors of green/gold like their sports teams and stuff)

File: style.css [hunk 1]
```diff
@@ -224,6 +224,21 @@
 	color: White;
 }
 
+.Australia .flip {
```
Comment (garykac) [inline style.css]: .flip is not used anywhere

**Comment by MrWeatherf** [comment]
Whoops, you're right. Thanks Gary!

**Comment by MrWeatherf** [comment]
Oops I accidentally closed this because I am not good with computers.

**Pull request merged**